### PR TITLE
feat: add get_network_id to nekoton. Expand github integration test time

### PR DESCRIPTION
**Pull request by Alex-A4**
- [x] Add `getNetworkId` to transport
- [x] Set github integration_test action timeout to 360 (default value)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -126,6 +126,7 @@ jobs:
 
   ios_integration_test:
     runs-on: macos-latest
+    timeout-minutes: 600
```
Comment (nesquikm) [inline .github/workflows/build.yml:129]: 10 hours?! Whoa! I think default 360 will be enough :)

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -155,6 +156,7 @@ jobs:
 
   android_integration_test:
     runs-on: macos-latest
+    timeout-minutes: 600
```
Comment (nesquikm) [inline .github/workflows/build.yml:159]: Another mind-blowing 10 hours :)

File: .github/workflows/build.yml (hunk 1 above)
Comment (nesquikm) [reply .github/workflows/build.yml:129]: We have "TimeoutException after 0:12:00.000000: Test timed out after 12 minutes." exception, that points us to completely separated problem:
https://github.com/flutter/flutter/issues/103064

**Review by nesquikm — CHANGES_REQUESTED** [review]
IMHO we should remove 10-hours "timeout-minutes" here and do some research about that 12-min timeouts.

File: .github/workflows/build.yml (hunk 1 above)
Comment (Alex-A4) [reply .github/workflows/build.yml]: -

File: .github/workflows/build.yml (hunk 2 above)
Comment (Alex-A4) [reply .github/workflows/build.yml]: -

**Review by nesquikm — APPROVED** [review]

**Pull request merged**